src/index.html

```diff
@@ -9,10 +10,52 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
```

Comment:
Keep your attributes correctly formatted. Each attribute should start on a new line with 2-space indentation related to the tag.